web/src/main/java/se/inera/intyg/webcert/web/web/controller/testability/facade/dto/CertificateType.java

```diff
@@ -30,6 +30,10 @@ public class CertificateType {
     private List<CertificateStatus> statuses;
     private List<CreateCertificateFillType> fillType;
 
+    public CertificateType() {
+        // For deserialization
```

Comment:
I think the comment is redundant... it can be used for other purposes as it now exists :-)

Reply:
done